acceptance/tests/sameness/sameness_test.go

```diff
@@ -127,19 +136,19 @@ func TestFailover_Connect(t *testing.T) {
 				+-------------------------------------------+
 			*/
 
-			members := map[string]*member{
-				keyPrimaryServer: {context: env.DefaultContext(t), hasServer: true},
-				keyPartition:     {context: env.Context(t, 1), hasServer: false},
-				keyPeer1:         {context: env.Context(t, 2), hasServer: true},
-				keyPeer2:         {context: env.Context(t, 3), hasServer: true},
+			testClusters := clusters{
+				keyCluster01a: {name: peerName1a, context: env.DefaultContext(t), hasServer: true, acceptors: []string{peerName2a, peerName3a}},
```

Comment:
Should this first keyCluster01a maybe have `partition: default` in it?  I believe that is implied but does it help with the readability?

Reply:
yeah, that's a good idea

Reply:
actually nevermind, I think the way I use the partition that if it's not blank `""` it causes issues in some places. So going to leave it as empty.